app/components/windows/ShareStream.tsx

```diff
@@ -127,7 +127,8 @@ export default class ShareStream extends TsxComponent<{ sharePageUrl: string }>
   }
 
   get sharePageUrl() {
-    return `facebook.com/${this.facebookService.state.activePage.name}-${this.facebookService.state.activePage.id}`;
+    return '';
+    // return `facebook.com/${this.facebookService.state.activePage.name}-${this.facebookService.state.activePage.id}`;
```

Comment:
i thought that we got rid of this page entirely?

Reply:
Yeah I'm also curious where this component is being used.  Can we delete?

Reply:
We can get rid of this component